tests/core/commands/test_init.py

```diff
@@ -96,6 +99,14 @@ def test_read_template_manifest():
         assert 'Basic Project 1' == manifest[0]['name']
         assert 'Description 2' == manifest[1]['description']
 
+        variables1 = manifest[0]['variables']
+        variables2 = manifest[1]['variables']
+        assert 0 == len(variables1)
+        assert 1 == len(variables2)
+        assert 1 == len(variables2.keys())
+        assert 'custom' in variables2.keys()
+        assert 'Custom Value' == variables2['custom']
+
```

Comment:
What happens if user passes a non-existing parameter? We should have a test for that.

Reply:
This is currently tested in `test_create_from_template` later in the file (line 170). We are passing parameters not listed in the expected parameters (`description`).

Anyway, that part is a bit confusing now but it will change soon since the last point of #1126 is about checking parameters validity.